content/efcore/getting-started/samples/configuring-deletion/deletion-configuration/LibraryContext.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+using REPLHelper;
+
+public class LibraryContext : DbContext
+{
+    public DbSet<Address> Addresses { get; set; }
+    public DbSet<Author> Authors { get; set; }
+    public DbSet<Book> Books { get; set; }
+    public DbSet<CheckoutRecord> CheckoutRecords { get; set; }
+    public DbSet<Reader> Readers { get; set; }
+
+    protected override void OnModelCreating(ModelBuilder modelBuilder)
+    {
+        modelBuilder.Entity<CheckoutRecord>()
```

Comment:
Should be setting behavior of `Reader` in relation to `Address`, not `CheckoutRecord`